### PR TITLE
Add riblee handwired split keyboard

### PR DESCRIPTION
**Pull request by riblee**
## Description

This PR adds support for a handwired 12X5 split keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/riblee_split/info.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B10", "B1", "B0", "A7", "A6", "A5"],
+        "rows": ["A4", "A3", "A2", "A1", "A0"]
+    },
+    "processor": "STM32F411",
```
Comment (dunk2k) [inline keyboards/handwired/riblee_split/info.json:19]: `processor` defined by `development_board`
```suggestion
```

File: keyboards/handwired/riblee_split/info.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        "device_version": "1.0.0",
+        "vid": "0xFEED",
+        "pid": "0x2B2B"
+    },
+    "split": {
+        "enabled": true,
+        "transport": {
+            "sync_matrix_state": true
+        }
+    },
```
Comment (dunk2k) [inline keyboards/handwired/riblee_split/info.json:32]:
```suggestion
    },
    "community_layouts": ["ortho_5x12"],
```

File: keyboards/handwired/riblee_split/info.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                {"matrix": [9, 0], "x": 7, "y": 4},
+                {"matrix": [9, 1], "x": 8, "y": 4},
+                {"matrix": [9, 2], "x": 9, "y": 4},
+                {"matrix": [9, 3], "x": 10, "y": 4},
+                {"matrix": [9, 4], "x": 11, "y": 4},
+                {"matrix": [9, 5], "x": 12, "y": 4}
+            ]
+        }
+    }
+}
```
Comment (dunk2k) [inline keyboards/handwired/riblee_split/info.json:108]: add EoF
```suggestion
}

```

File: keyboards/handwired/riblee_split/halconf.h [hunk 4]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 Daniel Reibl (@riblee)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define HAL_USE_SERIAL TRUE
+
+#include_next <halconf.h>
```
Comment (dunk2k) [inline keyboards/handwired/riblee_split/halconf.h:8]: add EOF
```suggestion
#include_next <halconf.h>

```

File: keyboards/handwired/riblee_split/mcuconf.h [hunk 5]
```diff
@@ -0,0 +1,7 @@
+// Copyright 2023 Daniel Reibl (@riblee)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include_next <mcuconf.h>
+
+#undef STM32_SERIAL_USE_USART1
+#define STM32_SERIAL_USE_USART1 TRUE
```
Comment (dunk2k) [inline keyboards/handwired/riblee_split/mcuconf.h:7]: add EoF
```suggestion
#define STM32_SERIAL_USE_USART1 TRUE

```

File: keyboards/handwired/riblee_split/rules.mk [hunk 6]
```diff
@@ -0,0 +1 @@
+SERIAL_DRIVER = usart
```
Comment (dunk2k) [inline keyboards/handwired/riblee_split/rules.mk:1]: add EoF
```suggestion
SERIAL_DRIVER = usart

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**